src/foam/comics/DAOControllerView.js

```diff
@@ -62,7 +62,8 @@ foam.CLASS({
     }
 
     ^container {
-      display: flex;
+      display: grid;
```

Comment:
The changes in this file are required for this to work in Safari. There's a bug in Safari where `position: sticky` elements won't display correctly if they have a `display: flex` ancestor.